vm_data.py-dist

```diff
@@ -1,7 +1,33 @@
 # All values below are left empty on purpose, for you to fill them with your VM references.
 # The VM references will be passed over by jobs.py to pytest through pytest's --vm parameter.
-# Consult README.md for accepted values.
-# Example value: "http://somewhere/images/small_vm.xva"
+
+# The base URL for your VM references must be specified as:
+XVA_LOCATION="http://somewhere.domain/"
```

Comment:
I now remember why `XVA_LOCATION` was not defined in `vm_data.py-dist`: because I wanted to give the liberty to just make the file a simple dict, with all URLs repeated if needed, and maybe using XVAs downloaded from various places.

It's not a strong requirement, though, so if it makes it easier for us, I'm OK with enforcing the use of XVA_LOCATION.

Reply:
So now XVA_LOCATION is used only for XVAs that are not URLs

Reply:
I would have let XVA_LOCATION be specific to our instance of vm_data.py, and only define a dict in the `vm_data.py-dist`, but I'm not opposed to your change either.

Reply:
Oh, actually, there is an issue with this change. Values can also be VM UUIDs.

Reply:
So, in my opinion, it's best not to add any logic to `vm_data.py-dist`. `jobs.py` must expect a dict of URLs or UUIDs, and that's it. Defining a prefix for URLs is up to the user who makes their own `vm_data.py`.

Reply:
As discussed, our version of `vm_data.py` needs some code to prepend `XVA_LOCATION` to all VM entries. And now, host version is matched using a regular expression,